RecoLocalCalo/HGCalRecProducers/plugins/HGCalLayerClusterProducer.cc

```diff
@@ -97,18 +93,46 @@ HGCalClusterProducer::HGCalClusterProducer(const edm::ParameterSet &ps) :
     algo = std::make_unique<HGCalImagingAlgo>(vecDeltas, kappa, ecut, algoId, dependSensor, dEdXweights, thicknessCorrection, fcPerMip, fcPerEle, nonAgedNoises, noiseMip, verbosity);
   }
 
-  auto sumes = consumesCollector();
-
-  multicluster_algo = std::make_unique<HGCal3DClustering>(ps, sumes, multicluster_radii, minClusters);
 
   produces<std::vector<reco::BasicCluster> >();
   produces<std::vector<reco::BasicCluster> >("sharing");
 
-  produces<std::vector<reco::HGCalMultiCluster> >();
-  produces<std::vector<reco::HGCalMultiCluster> >("sharing");
 }
 
-void HGCalClusterProducer::produce(edm::Event& evt,
+
+void HGCalLayerClusterProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  // hgcalLayerClusters
+  edm::ParameterSetDescription desc;
+  desc.add<std::string>("detector", "all");
+  desc.add<bool>("doSharing", false);
+  desc.add<std::vector<double>>("deltac", {
+    2.0,
+    5.0,
```

Comment:
in the old config file, it was 2,2,5. I think this could be the cause of the changes in the latest round of comparisons.

Reply:
thanks Kevin for spotting the difference!